Sources/TokamakCore/Views/Selectors/DatePicker.swift

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Emil Pedersen on 2021-03-26.
+//
+
+import struct Foundation.Date
+
+/// A control for selecting a value from a bounded linear range of values.
```

Comment:
This seems like a description of a Slider. Maybe try
```suggestion
/// A control for selecting a `Date`.
```
or
```suggestion
/// A control for selecting a `Date` from a bounded linear range of dates.
```